src/databricks/labs/remorph/reconcile/recon_capture.py

```diff
@@ -220,16 +223,49 @@ def _insert_into_main_table(
         )
         _write_df_to_delta(df, f"{self._db_prefix}.{_RECON_TABLE_NAME}")
 
+    @classmethod
+    def _apply_threshold(
+        cls, data_reconcile_output: DataReconcileOutput, table_conf: Table, record_count: ReconcileRecordCount
+    ):
+        # if the mismatch count is 0 then no need of checking bounds.
+        if data_reconcile_output.mismatch_count == 0:
+            return False
+        # pull out table thresholds
+        thresholds: list[TableThresholds] = (
+            [threshold for threshold in table_conf.table_thresholds if threshold.model == TableThresholdModel.MISMATCH]
+            if table_conf.table_thresholds
+            else []
+        )
+        res = None
+        if thresholds:
+            for threshold in thresholds:
+                mode = threshold.get_mode()
+                lower_bound = int(threshold.lower_bound.replace("%", ""))
+                upper_bound = int(threshold.upper_bound.replace("%", ""))
+                if mode == "absolute":
```

Comment:
Can we re-use `_build_threshold_absolute_case` function in Threshold_query?

Reply:
No, The logics are different.